rest/handler/tracinghandler.go

```diff
@@ -13,15 +12,26 @@ import (
 	oteltrace "go.opentelemetry.io/otel/trace"
 )
 
-var notTracingSpans sync.Map
+type (
+	// TracingOptions is TracingHandler options.
+	TracingOptions struct {
+		traceIgnorePaths []string
+	}
 
-// DontTraceSpan disable tracing for the specified span name.
-func DontTraceSpan(spanName string) {
-	notTracingSpans.Store(spanName, lang.Placeholder)
-}
+	// TracingOption defines the method to customize an tracingOptions.
+	TracingOption func(options *TracingOptions)
+)
 
 // TracingHandler return a middleware that process the opentelemetry.
-func TracingHandler(serviceName, path string) func(http.Handler) http.Handler {
+func TracingHandler(serviceName, path string, opts ...TracingOption) func(http.Handler) http.Handler {
+	var tracingOptions TracingOptions
+	for _, opt := range opts {
+		opt(&tracingOptions)
+	}
+
+	ignorePaths := collection.NewSet()
```

Comment:
```go
if len(tracingOptions.traceIgnorePaths) == 0 {
    return func(next http.Handler) http.Handler {
        return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
            next.ServeHTTP(w, r)
        }
    }
}
```

Reply:
`len(tracingOptions.traceIgnorePaths) == 0` means use tracing middleware to all paths.